source/scatter.f90

```diff
@@ -13,488 +13,257 @@ module scatter
+    setFields(7)  = h5_dataField(name="T",       type=h5_typeReal)
+    setFields(8)  = h5_dataField(name="THETA",   type=h5_typeReal)
+    setFields(9)  = h5_dataField(name="PHI",     type=h5_typeReal)
+    setFields(10) = h5_dataField(name="dE/E",    type=h5_typeReal)
+    setFields(11) = h5_dataField(name="dP/P",    type=h5_typeReal)
+    setFields(12) = h5_dataField(name="DENSITY", type=h5_typeReal)
+    setFields(13) = h5_dataField(name="PROB",    type=h5_typeReal)
+    setFields(14) = h5_dataField(name="STATCORR",type=h5_typeReal)
 
-    call h5_initForScatter()
     call h5_createFormat("scatter_log_fmt", setFields, scatter_logFormat)
     call h5_createDataSet("scatter_log", h5_scatID, scatter_logFormat, scatter_logDataSet)
+    block
+      character(len=:), allocatable :: colNames(:)
+      character(len=:), allocatable :: colUnits(:)
+      logical spErr
+      integer nSplit
+      call chr_split("ID turn bez generator process lost t dE/E dP/P theta phi density prob stat_corr",colNames,nSplit,spErr)
+      call chr_split("1 1 text text text 1 MeV^2 1 1 mrad rad mb^-1 1 1",colUnits,nSplit,spErr)
+      call h5_writeDataSetAttr(scatter_logDataSet,"colNames",colNames)
+      call h5_writeDataSetAttr(scatter_logDataSet,"colUnits",colUnits)
+    end block
+
+    deallocate(setFields)
+
+    ! Scatter Summary
+    allocate(setFields(10))
+
+    setFields(1)  = h5_dataField(name="TURN",      type=h5_typeInt)
+    setFields(2)  = h5_dataField(name="NAPX",      type=h5_typeInt)
+    setFields(3)  = h5_dataField(name="BEZ",       type=h5_typeChar, size=mNameLen)
+    setFields(4)  = h5_dataField(name="GEN",       type=h5_typeChar, size=mNameLen)
+    setFields(5)  = h5_dataField(name="PROCESS",   type=h5_typeChar, size=8)
+    setFields(6)  = h5_dataField(name="NSCAT",     type=h5_typeInt)
+    setFields(7)  = h5_dataField(name="NLOST",     type=h5_typeInt)
+    setFields(8)  = h5_dataField(name="SCATRATIO", type=h5_typeReal)
+    setFields(9)  = h5_dataField(name="CROSSSEC",  type=h5_typeReal)
+    setFields(10) = h5_dataField(name="SCALING",   type=h5_typeReal)
+
+    call h5_createFormat("scatter_summary_fmt", setFields, scatter_sumFormat)
+    call h5_createDataSet("summary", h5_scatID, scatter_sumFormat, scatter_sumDataSet)
+    block
+      character(len=:), allocatable :: colNames(:)
+      character(len=:), allocatable :: colUnits(:)
+      logical spErr
+      integer nSplit
+      call chr_split("turn napx bez generator process nScat nLost scatRat scaling",colNames,nSplit,spErr)
+      call chr_split("1 1 text text text 1 1 1 1",colUnits,nSplit,spErr)
+      call h5_writeDataSetAttr(scatter_sumDataSet,"colNames",colNames)
+      call h5_writeDataSetAttr(scatter_sumDataSet,"colUnits",colUnits)
+    end block
+
+    deallocate(setFields)
 
     ! Write Attributes
-    call h5_writeAttr(h5_scatID,"SEED",(/scatter_seed1,scatter_seed2/))
+    call h5_writeAttr(h5_scatID,"SEED",scatter_seed1)
```

Comment:
Why only writing one seed? If on purpose, a comment may be nice.

Reply:
Because the SCATTER block takes one seed.

Reply:
Ah, one input seed that gets printed and then converted to two internal seeds? Got it.